File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/inventory/services/StockService.java
package com.metaengine.demo.inventory.services;

import com.metaengine.demo.inventory.aggregates.StockItem;
import java.util.List;

/** Application service exposing stock item operations. */
public class StockService {
    /** Create a stock item from the provided partial input. */
    public StockItem create(StockItem input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a stock item by id. Returns null when not found. */
    public StockItem findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} stock items. */
    public List<StockItem> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a stock item by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/inventory/services/stock_service.py
"""StockService for the inventory domain."""
from typing import Optional

from metaengine_demo.inventory.aggregates.stock_item import StockItem


class StockService:
    """StockService application service."""

    def create(self, input: Optional[StockItem]) -> StockItem:
        """Create a new stock item."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[StockItem]:
        """Find a stock item by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[StockItem]:
        """List stock items up to a limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a stock item by id."""
        raise NotImplementedError("not implemented")
